hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/RolloutServiceContext.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ui.rollout.rollout;
+
+import org.eclipse.hawkbit.repository.EntityFactory;
+import org.eclipse.hawkbit.repository.QuotaManagement;
+import org.eclipse.hawkbit.repository.RolloutGroupManagement;
+import org.eclipse.hawkbit.repository.RolloutManagement;
+import org.eclipse.hawkbit.repository.TargetFilterQueryManagement;
+import org.eclipse.hawkbit.repository.TargetManagement;
+import org.eclipse.hawkbit.repository.TargetQueryExecutionManagement;
+import org.eclipse.hawkbit.repository.TenantConfigurationManagement;
+
+public class RolloutServiceContext {
+    public final RolloutManagement rolloutManagement;
```

Comment:
use getters instead of public final fields, to stick to the convention

Reply:
There is one time where final makes sense ;-) ....

Reply:
I change it - but to make it clear there are no conventions for this aslo not in your coding guidelines - get/setter are for "java-beans" this is a struct-style class not a bean.